relentless/potential/pair.py

```diff
@@ -156,32 +156,151 @@ def _derivative(self, param, r, epsilon, sigma, **params):
             d = d.item()
         return d
 
-""" TODO: these potentials need updating
 class Yukawa(PairPotential):
+    """Yukawa pair potential.
+
+    .. math::
+
+    u(r) = \varepsilon\frac{{e}^{{-\kappa{r}}}}{{r}}
```

Comment:
This needs another indent and can be written like:
```suggestion
    u(r) = \varepsilon\frac{e^{-\kappa r}}{r}
```